inc/classes/subscriber/Tools/class-detect-missing-tags-subscriber.php

```diff
@@ -123,4 +127,24 @@ public function rocket_notice_missing_tags() {
 			]
 		);
 	}
+
+	/**
+	 * Get the request URI.
+	 *
+	 * @since  3.4.2
+	 * @author Soponar Cristina
+	 *
+	 * @return string
+	 */
+	public function get_raw_request_uri() {
+		if ( ! isset( $_SERVER['REQUEST_URI'] ) ) {
+			return '';
+		}
+
+		if ( '' === $_SERVER['REQUEST_URI'] ) {
+			return '';
+		}
+
+		return '/' . ltrim( $_SERVER['REQUEST_URI'], '/' );
```

Comment:
I found nothing about escaping within logs, so I guess we should do it here. Also, `wp_unslash()`.
```suggestion
		return '/' . esc_html( ltrim( wp_unslash( $_SERVER['REQUEST_URI'] ), '/' ) );
```